Incompatible with protobuf 30.0

Hello hello, Arch Linux packager here 👋🏻 

We have recently started the [protobuf 30.0](https://archlinux.org/todo/protobuf-300/) rebuild and the astroid package is one of the holdups that came up along the way due to the following compiler error:

```
/build/astroid/src/astroid-0.16/src/modes/thread_view/webextension/tvextension.cc:2230:102: error: conversion from ‘google::protobuf::internal::DescriptorStringView’ {aka ‘std::basic_string_view<char>’} to non-scalar type ‘std::string’ {aka ‘std::__cxx11::basic_string<char>’} requested
 2230 |   std::string _t = AstroidMessages::Navigate_Type_descriptor ()->FindValueByNumber (n.type ())->name ();
      |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
/build/astroid/src/astroid-0.16/src/modes/thread_view/webextension/tvextension.cc:2233:60: warning: ‘WebKitDOMDocument* webkit_web_page_get_dom_document(WebKitWebPage*)’ is deprecated [-Wdeprecated-declarations]
 2233 |   WebKitDOMDocument * d = webkit_web_page_get_dom_document (page);
      |                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~
/usr/include/webkitgtk-4.0/webkit/WebKitWebPage.h:78:1: note: declared here
   78 | webkit_web_page_get_dom_document            (WebKitWebPage *web_page);
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/astroid/src/astroid-0.16/src/modes/thread_view/webextension/tvextension.cc:2234:65: warning: ‘WebKitDOMDOMWindow* webkit_dom_document_get_default_view(WebKitDOMDocument*)’ is deprecated [-Wdeprecated-declarations]
 2234 |   WebKitDOMDOMWindow * w = webkit_dom_document_get_default_view (d);
```

I think this is due to a few string interface changes, see for example the changes from https://github.com/protobuf-c/protobuf-c/pull/762 🤗

--- a/src/modes/thread_view/webextension/tvextension.cc
+++ b/src/modes/thread_view/webextension/tvextension.cc
@@ -2272,7 +2272,7 @@ void AstroidExtension::scroll_to_element (ustring eid) {
 }
 
 void AstroidExtension::handle_navigate (AstroidMessages::Navigate &n) {
-  std::string _t = AstroidMessages::Navigate_Type_descriptor ()->FindValueByNumber (n.type ())->name ();
+  std::string _t { AstroidMessages::Navigate_Type_descriptor ()->FindValueByNumber (n.type ())->name () };
   LOG (debug) << "navigating, type: " << _t;
 
   WebKitDOMDocument * d = webkit_web_page_get_dom_document (page);

